[Image Component] image sizes property becomes 100vw by default (#20067)
vercel/next.js: docs/api-reference/next/image.md

### sizes

A string mapping media queries to device sizes. Defaults to `100vw`.

> We recommend setting `sizes`, as it helps the browser choose the right image size to use.

Reviewer: ```suggestion
We recommend setting `sizes` when `layout="responsive"` and your image will not be the same width as the viewport.
```
